### PR TITLE
AR prototype merge

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity/issues/733

*Description of changes:*
Updated coops-ui Home, AnnualReport, AGMDate, Directors etc to update layout and CSS to closely match Jeremy's latest approved prototype.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by katiemcgoff** [comment]
Two notes so far:
1. margins are a bit out on containers - I don't think you'll need the ar-step-1-container type of Ids, and they are what is causing the margins.
2. change button style in Registered Office section to updated style from AnnualReportAlt version.

image

**Comment by katiemcgoff** [comment]
Re. squashing, no need to do that until review is complete - the reviewer (me) does that right before merging.

File: coops-ui/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@babel/polyfill": "^7.0.0-rc.1",
     "axios": "^0.18.0",
+    "moment": "^2.24.0",
```
Comment (katiemcgoff) [inline coops-ui/package.json]: Need to remove this and the file it's used in - ~ARFilingDates.vue

Comment (severinbeauvais) [reply coops-ui/package.json]: OK.

**Comment by severinbeauvais** [comment]
All changes have been made as requested.

**Comment by katiemcgoff** [comment]
Linting errors. Use the lint command (see screenshot) to fix whatever it can and see errors. Many are line-length warnings, and some are real lint errors (obviously not functional errors because it all runs fine). Our team standard is zero lint errors or warnings before it can be approved.

image

**Comment by severinbeauvais** [comment]
Looks OK to me. It was probably the ~ (prototype) files, which I have since deleted.
lint-no-fix

**Pull request merged**